src/utils/mev.ts

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios'
+import { ethers } from 'ethers'
+const logger = require('../services/logger')
+
+const bevUrl = 'https://bev-relay.kolibr.io'
+
+export function isEthSendMethod(rawData) {
+  if (rawData instanceof Array) {
+    return false
+  }
+  if (rawData.method === 'eth_sendRawTransaction') {
+    return true
+  }
+  return false
+}
+
+export async function checkBevRequest(requestId, txRawData, broadcasterAddress) {
+  try {
+    const res = await axios.post(bevUrl, {
+      jsonrpc: '2.0',
+      id: requestId,
+      method: 'check_bev',
+      params: {
+        tx_raw_data: txRawData,
+        broadcaster_address: broadcasterAddress,
+        wait_searcher_ms: 500,
+      },
+    })
+    if (res?.data?.result?.broadcaster_diff) {
+      return res.data.result
+    }
+  } catch (e) {
+    console.log(e.message)
+  }
+
+  return false
+}
+
+export async function submitBevRequest(requestId, txRawData, bundleHash) {
+  try {
+    const res = await axios.post(bevUrl, {
+      jsonrpc: '2.0',
+      id: requestId,
+      method: 'submit_bev',
+      params: {
+        tx_raw_data: txRawData,
+        bundle_hash: bundleHash,
+      },
+    })
+    return res?.data?.result
+  } catch (e) {
+    console.log(e.message)
+  }
+
+  return false
+}
+
+export async function handleBEVRequest(id, parsedRawData, address) {
+  // store address in Github secrets
+  const checkBevResult = await checkBevRequest(id, parsedRawData, address)
+  logger.log('Check BEV result:', checkBevResult)
```

Comment:
the first argument of `logger.log` must be the type of log (as in `debug, infro, warn, errror`)